server/api/admin_test.go

```diff
@@ -91,10 +91,10 @@ func (s *testAdminSuite) TestDropRegion(c *C) {
 
 func (s *testAdminSuite) TestPersistFile(c *C) {
 	data := []byte("#!/bin/sh\nrm -rf /")
-	err := postJSON(testDialClient, s.urlPrefix+"/admin/persist-file/fun.sh", data)
-	c.Assert(err, NotNil)
+	err := tu.CheckPostJSON(testDialClient, s.urlPrefix+"/admin/persist-file/fun.sh", data, tu.StatusNotOK(c))
+	c.Assert(err, IsNil)
```

Comment:
Why change the assert?

Reply:
> Why change the assert?

`postJSON` will return err if `StatusCode` is not equal 200. But now this check does not exist in `CheckPostJSON `.  If need, you can add check action `StatusNotOK`.